source/docs/guides/secure-wordpress-or-drupal-with-sucuri.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Secure WordPress or Drupal Sites with Sucuri Cloud Proxy
+description: Detailed information on using Sucuri Cloud Proxy to protect against malware, hackers, and blacklists.
+category:
+  - developing
+authors:
+  - atjuch
+date: 7/28/2015
+---
+With malicious attacks gaining popularity across the internet, it is becoming increasingly important to secure your site. The intent of these attacks vary, ranging from service interruption to malware injections, and more. In this guide, we will set up a Cloud Proxy service to protect your website and stay ahead of potential attacks.
+## Why Sucuri?
+
+[Sucuri](https://sucuri.net) is one of the leading website security companies in the world. They offer a variety of paid and online tools ranging from [Website AntiVirus](https://sucuri.net/website-antivirus/), [Cloud Proxy (Firewall)](https://sucuri.net/website-firewall/), [Free Website Security Scan](https://sitecheck.sucuri.net/), and more.
+Whether you're looking to secure a personal site or an enterprise client, Sucuri has you covered with 24/7 ticket support and incredible response times.
+
+## Prerequisites
+
+### Direct Your Domain to Pantheon
+In order to add Sucuri Cloud Proxy services, you must already have a domain directed to Pantheon and added to the Live environment. For more instructions, please see [DNS Records for Directing Your Domain to Your Pantheon Site](/source/docs/articles/sites/domains/dns-records-for-directing-your-domain-to-your-pantheon-site/).
+
+## Create a Sucuri Account
+
+Get started by [selecting a plan](https://sucuri.net/website-firewall/signup) that fits your needs. Once a plan is selected, you will be instructed to create your Sucuri account and select a payment method.
+
+![Sucuri sign up form](/source/docs/assets/images/sucuri-cloud-proxy-signup.png)​
+
+## Login to your Sucuri Account
+
+After receiving your confirmation email, sign in to your [Sucuri account](https://login.sucuri.net/login/). From within the Dashboard, click on **CloudProxy Website Firewall**.
+
+![Sucuri Account Settings](/source/docs/assets/images/sucuri-account-settings.png)
+
+## Add Your Site
+
+Enter your domain and click **Add Site** to enable CloudProxy. You can add it with or without www, Sucuri will remove the www automatically.
+
+![Sucuri Add Site](/source/docs/assets/images/sucuri-add-website.png)
+
+## Verify Domain Name
+
+You will see a pop-up that asks you to verify the domain name and server location. Pantheon sites are served from North America, but you can select a location that represents where you live.
+
+Indicate whether you are currently under a DDoS attack or if you would like to restrict your site's backend to whitelisted IPs. When you are ready, click **Proceed**
```

Comment:
Hi Rachel, we wouldn't need to do that since that is a major lock-down. I think the description you updated is great. If they want to read more about it or get help from Sucuri they can.